README.md

```diff
@@ -0,0 +1,20 @@
+# Release Information
+
+- **Version**: 1.0.0
+
+- **Certified**: Yes
+
+- **Publisher**: Fortinet  
+
+- **Compatibility**: 7.3.1 and later
+
+- **Applicable**: FortiAI Solution Pack
+
+# Overview
+
+The **FortiAI Configuration** widget helps to set up LLM Integration to use and Model to be used for different mode of FortiAI.
```

Comment:
The **FortiAI Configuration** widget helps to set up LLM Integration and Model to be used for a different mode of the FortiAI.